pages/general/index.mdx

```diff
@@ -1,9 +1,24 @@
 import IncompletePage from '@components/IncompletePage.tsx'
+import IncompleteSection from '@components/IncompleteSection.tsx'
+import Link from "next/link";
 
 # Willkommen
+Herzlich willkommen beim BWINF-Guide, einem Leitfaden, der dir mit Tutorials und ähnlichen Ressourcen während deiner Teilnahme am BWINF zur Seite steht. Dieser Guide wurde von ehemaligen Teilnehmern ins Leben gerufen und befindet sich derzeit noch in der Entwicklungsphase.
 
-Herzlich Willkommen auf dem BWINF Guide, ein Guide, welcher euch bei eurer Teilnahme am BWINF mit Tutorials und Ähnlichem unterstützen sollte. Dieser Guide wurde von ehemaligen Teilnehmern erstellt und ist derzeit noch in seinen Anfangsstadien. Falls euch ein Fehler in dem Guide auffallen sollte, oder ihr eine Idee für ein Thema habt, worüber ein Artikel geschrieben werden sollte, könnt ihr gerne ein Pull-Request auf Github erstellen.
+## Der Bundeswettbewerb Informatik
+<IncompleteSection />
 
-Zu beachten ist, dass dieser Guide nicht von dem offiziellen BWINF-Komitee erstellt wurde, und demnach kein Curriculum o. Ä. ist. Dies bedeutet, dass in zukünftigen Wettbewerbsrunden Aufgaben vorkommen können, die bisher im Guide noch nicht behandelt wurden. Wir geben jedoch unser bestes, den Guide so aktuell wie möglich zu halten. Selbstverständlich werden hier keine Lösungen zu aktuell laufenden Wettbewerbsrunden veröffentlicht.
+## Ziel des Guides
+<IncompleteSection />
 
+## Kurzübersicht
+<IncompleteSection />
+
+## Code of Conduct
+<IncompleteSection />
```

Comment:
gehört nicht zu Willkommen, eher in > Beitragen

Reply:
Beitragen wird sich aber vermutlich ein Großteil der User nicht durchlesen obwohl das für jeden denk ich wichtig ist. Da kann man so Sachen, wie Wie man mit den Ressourcen umgehen darf und das Freundliches Verhalten erwartet wird. In Beitragen ließt sich das keiner durch, trotzdem können wir das gern in eine andere Section bewegen.

Reply:
wir sind hier auch kein Forum, insofern gibt es eigentlich nirgendwo wo man sich "verhalten" muss, geschweige "Freundliches Verhalten"

Man kommt erst mit uns in Interaktion, wenn man was mit dem Repo machen muss -> Beitragen
oder in Discord, dann gehört der Teil in Rules Kanal in Discord

Reply:
Stimmt und sonst kann man das später in die AGB packen wenn man ein Login System oder so hat